umm-spec-lib/src/cmr/umm_spec/migration/version/service.clj

```diff
@@ -551,3 +593,20 @@
 (defmethod interface/migrate-umm-version [:service "1.3.1" "1.3"]
   [context s & _]
   (update-service-options-1_3_1->1_3 s))
+
+(defmethod interface/migrate-umm-version [:service "1.3.1" "1.3.2"]
```

Comment:
This file is quite long. We should probably start to break up the code for each version into its own file or something.  Whether we handle it in a separate ticket or not doesn't matter so much to me, but it feels worthwhile

Reply:
I can break it up in the next iteration coming later today.